source/server/options_impl.cc

```diff
@@ -280,7 +280,8 @@ Server::CommandLineOptionsPtr OptionsImpl::toCommandLineOptions() const {
 
 OptionsImpl::OptionsImpl(const std::string& service_cluster, const std::string& service_node,
                          const std::string& service_zone, spdlog::level::level_enum log_level)
-    : base_id_(0u), concurrency_(1u), config_path_(""), config_yaml_(""),
+    : base_id_(0u), concurrency_(1u), config_path_(""),
+      config_proto_(envoy::config::bootstrap::v2::Bootstrap()), config_yaml_(""),
```

Comment:
nit: this initializer is not needed (default init is the same)

Reply:
Fixed.